docs/sdk/examples/pastefromgoogledocs.html

```diff
@@ -128,7 +128,7 @@ <h1>Pasting content from Google Docs <a class="documentation" href="https://cked
 
 					&lt;p&gt;&nbsp;&lt;/p&gt;
 
-					&lt;p style="text-align:justify"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;The project that you participated in is of utmost importance to the future success of our platform. We are very proud to share that the CKEditor implementation was a huge success and brought congratulations from both the key Stakeholders and the Customers:&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+					&lt;p style="text-align:justify"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;The project that you participated in is of utmost importance to the future success of our platform. We are very proud to share that the CKEditor 4 implementation was a huge success and brought congratulations from both the key Stakeholders and the Customers:&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
```

Comment:
```suggestion
					&lt;p style="text-align:justify"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;The project that you participated in is of utmost importance to the future success of our platform. We are very proud to share that the CKEditor implementation was a huge success and brought congratulations from both the key Stakeholders and the Customers:&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
```

This content mimics the attached sample file so let's leave it as is.